actions/channel_queries.ts

```diff
@@ -0,0 +1,263 @@
+    nextChannels = 'gqlWebNextChannels',
+    nextTeamChannels = 'gqlWebNextTeamsChannels',
+    nextChannelMembers = 'gqlWebNextChannelMembers',
+    nextTeamChannelMembers = 'gqlWebNextTeamsChannelMembers'
+}
+
+type Cursor = {
+    cursor: string;
+}
+
+export type ChannelsQueryResponseType = {
+    data: {
+        channels: Array<Omit<ServerChannel, 'team_id'> & {
+            team: Team;
+        } & Cursor>;
+    };
+};
+
+export type ChannelMembersQueryResponseType = {
+    data: {
+        channelMembers: Array<Omit<ChannelMembership, 'channel_id | user_id | roles | post_root_id'> & {
+            channel: ServerChannel;
+            roles: Role[];
+        } & Cursor>;
+    };
+};
+
+export type ChannelsAndChannelMembersQueryResponseType = {
+    data: {
+        channels: ChannelsQueryResponseType['data']['channels'];
+        channelMembers: ChannelMembersQueryResponseType['data']['channelMembers'];
+    };
+}
+
+function makeChannelsAndMembersQueryString(
+    operationName: ChannelQueriessOperationNames,
+    teamId: Team['id'] = '',
+) {
+    return `
+    query ${operationName} {
+        channels(userId: "me", teamId: "${teamId}", first: ${CHANNELS_MAX_PER_PAGE}) {
+          id
+          create_at: createAt
+          update_at: updateAt
+          delete_at: deleteAt
+          team {
+            id
+          }
+          type
+          display_name: displayName
+          name
+          header
+          purpose
+          last_post_at: lastPostAt
+          last_root_post_at: lastRootPostAt
+          total_msg_count: totalMsgCount
+          total_msg_count_root: totalMsgCountRoot
+          creator_id: creatorId
+          scheme_id: schemeId
+          group_constrained: groupConstrained
+          shared
+          props
+          policy_id: policyId
+          cursor
+        }
+        channelMembers(userId: "me", teamId: "${teamId}", first: ${CHANNEL_MEMBERS_MAX_PER_PAGE}) {
+          cursor
+          channel {
+            id
+          }
+          roles {
+            id
+            name
+            permissions
+          }
+          last_viewed_at: lastViewedAt
+          msg_count: msgCount
+          msg_count_root: msgCountRoot
+          mention_count: mentionCount
+          mention_count_root: mentionCountRoot
+          notify_props: notifyProps
+          last_update_at: lastUpdateAt
+          scheme_admin: schemeAdmin
+          scheme_user: schemeUser
+        }
+      }
+`;
+}
+
+export function getAllChannelsAndMembersQueryString() {
+    return JSON.stringify({
+        query: makeChannelsAndMembersQueryString(ChannelQueriessOperationNames.allChannelAndMembers),
+        operationName: ChannelQueriessOperationNames.allChannelAndMembers,
+    });
+}
+
+export function getTeamsChannelsAndMembersQueryString(teamId: Team['id']) {
+    return JSON.stringify({
+        query: makeChannelsAndMembersQueryString(ChannelQueriessOperationNames.teamsChannelAndMembers, teamId),
```

Comment:
There is no need to pass variables this way. Please use variable binding and pass the teamId separately: https://graphql.org/learn/queries/#variables

Reply:
added variable supports to the query. please check